src/domain/classes/LottoOrganizer.js

```diff
@@ -12,17 +12,14 @@ class LottoOrganizer {
   }
 
   #getWinningDetails(matchLottoNumberCount, matchBonusNumberCount) {
-    const matcherMap = new Map(Object.entries(LOTTO_MATCH_COUNT));
-    let rank = 'OTHERS';
+    const matcherArray = Array.from(Object.entries(LOTTO_MATCH_COUNT));
 
-    matcherMap.forEach((value, key) => {
+    const rank = matcherArray.find(([, value]) => {
       const { BASE_NUMBER, BONUS_NUMBER } = value;
-      if (matchLottoNumberCount === BASE_NUMBER && matchBonusNumberCount === BONUS_NUMBER) {
-        rank = key;
-      }
+      return matchLottoNumberCount === BASE_NUMBER && matchBonusNumberCount === BONUS_NUMBER;
     });
 
-    return rank;
+    return rank ? rank[0] : 'OTHERS';
```

Comment:
구성하신 Node 환경에서 지원이 될지 모르겠지만 이렇게도 가능합니다ㅎㅎ

```suggestion
    return rank.at(0) ?? 'OTHERS';
```

Reply:
오호..더 선언적인 문법같네요. 감사합니다!